spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java

```diff
@@ -33,6 +33,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
Okay, maybe we can catch up on gitter tomorrow and perhaps you can explain the fundamental things that I'm missing, as far as you'd like. :) I'll narrow down precisely which autoconfiguration class was stopping me from even starting my application. Thanks!